test/linter/test_yaml_actionlint.vader

```diff
@@ -1,9 +1,8 @@
 Before:
-  runtime! ale/handlers/actionlint.vim
+  call ale#assert#SetUpLinterTest('yaml', 'actionlint')
 
 After:
-  unlet! g:ale_yaml_actionlint_options
-  call ale#linter#Reset()
+  call ale#assert#TearDownLinterTest()
```

Comment:
I had to update those lines as the test would always fail when running `./run-tests test/linter/test_yaml_actionlint.vader`. Even on `master`. 

I took inspiration from all the other tests in the directory as it seems to be the proper way to setup/teardown the tests.